javascripts/discourse/initializers/tag-icons.js.es6

```diff
@@ -73,8 +73,29 @@ export default {
   name: "tag-icons",
 
   initialize() {
-    withPluginApi("0.8.31", (api) => {
+    withPluginApi("1.6.0", (api) => {
       api.replaceTagRenderer(iconTagRenderer);
+
+      if (api.registerCustomTagSectionLinkPrefixIcon) {
+        const tagIconList = settings.tag_icon_list.split("|");
+
+        tagIconList.forEach((tagIcon) => {
+          const [tagName, icon, color] = tagIcon.split(",");
+
+          if (tagName && icon) {
+            const opts ={
+              tagName,
+              prefixValue: icon
+            }
+
+            if (color) {
+              opts.prefixColor = color;
+            }
```

Comment:
nitpick, but the if is not really needed here.

```javascript
const opts = {
  tagName,
  prefixValue: icon,
  prefixColor: color
}
```

Would just work the same IMO, up to you.

Reply:
:+1: